lms/djangoapps/verify_student/views.py

```diff
@@ -1006,6 +1007,21 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt).all()
+        if checkpoints:
+            course_key = checkpoints[0].course_id
+            credit_requirement = CreditRequirement.get_course_requirement(
```

Comment:
As a general rule, please avoid using Django models from the `credit` app in other Django apps.  Instead, import `credit.api` and use those functions to update the status.